lib/core/src/lib/styles/_index.scss

```diff
@@ -72,9 +68,10 @@
         --adf-theme-background-selected-button-color: mat.get-color-from-palette($background, selected-button),
         --adf-theme-background-status-bar-color: mat.get-color-from-palette($background, status-bar),
         --adf-theme-background-unselected-chip-color: mat.get-color-from-palette($background, unselected-chip),
-
         // typography
         --theme-font-family: mat.font-family($typography),
+        --theme-font-weight: mat.font-weight($typography),
+        // not working - check it
```

Comment:
should we raise another ticket for it?

Reply:
Good point, it's a leftover from PoC that I forgot about, I have to check it. Thanks for pointing it out!